src/sdgio-edit.owl

```diff
@@ -158,6 +159,62 @@ Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020232>))
 Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020236>))
 Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020237>))
 Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020300>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020308>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020309>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020310>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020311>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020312>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020313>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020315>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020317>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020318>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020319>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020320>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020325>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020326>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020327>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020328>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020329>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020331>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020334>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020335>))
+Declaration(Class(<http://purl.unep.org/sdg/SDGIO_00020337>))
```

Comment:
@Mttbnchtt This class makes no sense to me. I think the language here is unintelligible and mostly "fluff". Is there anything more concrete you can find?

Reply:
I will delete it. I inserted a while ago to deal with CGIAR process to improve gender equity and youth inclusion. On Jun 4 2019 on Skype, discussing an earlier version of it, you suggested to keep it (but without knowing the definition). The definition seems roughly correct to me. The language of all-things-considered right tries to express the idea that fairness does not mean to respect every right (as rights may conflict), but to find an appropriate balance concerning which rights to respect and in which measure. However, the class is not doing any work now and it is probably better to delete. If you want me to reinstate it, I can do that and we can figure out how to define it.